sentences/sl/fan_HassTurnOn.yaml

```diff
@@ -2,7 +2,78 @@ language: sl
 intents:
   HassTurnOn:
     data:
-    - sentences: []
-      slots:
-        domain: fan
-        name: all
+      - sentences:
+          - <vključi> <name>
+      - sentences:
+          - <odpri> {name}
+        response: cover
+      - sentences:
+          - <odpri> {name} [v] {area}
+        response: cover_area
+
+      - sentences:
+          - <odpri> {name} (okno | okna) [v | na] {area}
+          - <odpri> [vsa] okna [v | na] {area}
+          - <odpri> {area} okno
+          - <odpri> {area} okna
+        response: windows_area
+        slots:
+          device_class:
+            - window
+          domain: cover
+
+      - sentences:
+          - <odpri> {name} (vrata) [v | na] {area}
+          - <odpri> [vsa] vrata [v | na] {area}
+          - <odpri> {area} vrata
```

Comment:
same here, you don`t have defined odpri, but you have defined open in expansion rules